jakartaee9/src/ejb-jar_4_0.xsds

```diff
@@ -1400,10 +1400,10 @@
 	      field
 	    - an optional declaration of the bean's environment 
 	      entries
-	    - an optional declaration of the bean's EJB 
+	    - an optional declaration of the bean's  
 	      references
 	    - an optional declaration of the bean's local 
-	      EJB references
+	      references
```

Comment:
Should be "the bean's local enterprise bean references".  This is about a bean having references to other beans